keyboards/meletrix/zoom65/zoom65.c

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2021 zoom
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "zoom65.h"
+
+static uint8_t encoder_state[ENCODERS] = {0};
```

Comment:
ideally, this should only be enabled for VIA.

Reply:
Added